apps/client/src/components/compounds/GroupView/ExploreGroupView/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { Button, Text } from 'components'
+import styles from './exploreGroup.module.scss'
+import frame from './images/Frame.svg'
+import rightless from './images/rightless.svg'
+import TableComponent from '../GroupTable'
+import CardComponent from '../GroupCard'
+
+const textArray = [
+  'Bros/Sis, no dull yourself for dis service year money mata Join Lagos Corp Members Thrift Saving Group sharp-sharp!We go secure your money, make am grow well-well, and support each other for dis money journey.Learn money sense, plan for your future, and enjoy the benefits of our community. No time to waste, come join us now! Contact [07067482633].Make we flex our money together!',
+]
+
+const cardData = [
+  {
+    label1: 'Contribution Amount',
+    value1: '₦ 500,000',
+    label2: 'Schedule',
+    value2: 'Daily',
+  },
+  {
+    label1: 'Contribution Timeline',
+    value1: '5 months',
+    label2: 'Estimated Collection',
+    value2: '₦ 2,000,000',
+  },
+  {
+    label1: 'Start Date',
+    value1: 'May 1, 2022',
+    label2: 'End Date',
+    value2: 'October 1, 2022',
+  },
+  {
+    label1: 'Available Slots',
+    value1: '2 and 6',
+    label2: 'Total Slots',
+    value2: '6',
+  },
+]
+
+const data = [
+  { slot: 'slot', name: 'name', performance: 'performance' },
+  { slot: '1', name: 'Oluwatomilola Eze', performance: '90.23%' },
+  { slot: '2', name: 'Mohammed Adebayo', performance: '45.50%' },
+  { slot: '3', name: 'Oluwadamilare Idowu', performance: '90.23%' },
+  { slot: '4', name: 'Harry Smith', performance: 'New user' },
+]
```

Comment:
When it's time to consume api, you would re write some implementation with this kind of data you are using